sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient_private.cs

```diff
@@ -23,7 +23,6 @@ public partial class ConfigurationClient
         private const string KeyQueryFilter = "key";
         private const string LabelQueryFilter = "label";
         private const string FieldsQueryFilter = "$select";
-        private const string ETag = "ETag";
```

Comment:
@annelo-msft is this something that will be used in the conditional get change?

Reply:
Nice catch.  No, that's only used in the HasChanged logic so it can go.